docs/remote/troubleshooting.md

```diff
@@ -516,8 +516,10 @@ The SSH extension provides a command for cleaning up the VS Code Server from the
 If you want to run these steps manually, or if the command isn't working for you, you can run a script like this:
 
 ```bash
-kill -9 `ps ax | grep "remoteExtensionHostAgent.js" | grep -v grep | awk '{print $1}'`
-kill -9 `ps ax | grep "watcherService" | grep -v grep | awk '{print $1}'`
+# Kill server processes and clean up related files
+kill -9 `ps ax | \grep vscode-server | \grep -v grep | awk '{print $1}'`
```

Comment:
```suggestion
kill -9 `ps ax | \grep "out/server-main.js" | \grep -v grep | awk '{print $1}'`
```

To be more consistent with what the Remote SSH extension looks for. This should yield the same result since all other server processes are children of `server-main`.

Reply:
Using that command does not find any processes for me (the server hash is `899d46d82c4c95423fb7e10e68eba52050e30ba3`, i.e. microsoft/vscode@899d46d82c4c95423fb7e10e68eba52050e30ba3)

I also noticed that on a multi-user system, both my command and yours return PIDs for other user's servers, which we don't want to (and won't be allowed to anyway) kill. So I think a better command would be:

~~~bash
kill -9 `ps aux | \grep vscode-server | \grep -v grep | \grep $USER | awk '{print $2}'`
~~~

Reply:
I just pushed an updated version with this change.

Reply:
You're on a very old version of VS Code `899d46d82c4c95423fb7e10e68eba52050e30ba3` is from 4 months ago. The USER grep is a good addition

Reply:
Unfortunately I can't upgrade at will on my work computer :) 

Since the docs at https://code.visualstudio.com/docs are not versioned, I think it would be nice if the command provided works not only for the most recent VSCode version, but for previous versions as well. That's why I would keep the `grep vscode-server`. What do you think ?

Reply:
Ah I see, okay yeah I agree that `vscode-server` works well here. 

> 
```
kill -9 `ps aux | \grep vscode-server | \grep -v grep | \grep $USER | awk '{print $2}'`
```

Looks good. Could you update your PR with this script?

Reply:
I updated it this morning: https://github.com/microsoft/vscode-docs/pull/5253#commits-pushed-2660759

Reply:
I don't see the grep for USER?

Reply:
woupsi, I went too fast this morning, you are right :) I just pushed a new version with the added `grep USER`

Reply:
Looks good to me!